library/common/extensions/cert_validator/platform_bridge/config.h

```diff
@@ -2,21 +2,28 @@
 
 #include "source/extensions/transport_sockets/tls/cert_validator/factory.h"
 
+#include "library/common/extensions/cert_validator/platform_bridge/platform_bridge.pb.h"
 #include "library/common/extensions/cert_validator/platform_bridge/platform_bridge_cert_validator.h"
 
 namespace Envoy {
 namespace Extensions {
 namespace TransportSockets {
 namespace Tls {
 
-class PlatformBridgeCertValidatorFactory : public CertValidatorFactory {
+class PlatformBridgeCertValidatorFactory : public CertValidatorFactory,
+                                           public Config::TypedFactory {
 public:
   CertValidatorPtr createCertValidator(const Envoy::Ssl::CertificateValidationContextConfig* config,
                                        SslStats& stats, TimeSource& time_source) override;
 
   std::string name() const override {
     return "envoy_mobile.cert_validator.platform_bridge_cert_validator";
   }
+  ProtobufTypes::MessagePtr createEmptyConfigProto() override {
+    return std::make_unique<
+        envoy_mobile::extensions::cert_validator::platform_bridge::PlatformBridgeCertValidator>();
+  }
+  std::string category() const override { return "envoy.tls.cert_validator"; }
```

Comment:
In envoy_build_config/extensions_build_config.bzl, the extension shows up as "envoy_mobile.cert_validator.platform_bridge_cert_validator" (which matches name()). Should these be made consistent? How 'bout we change the name to ""envoy.tls.cert_validator.platform_bridge"? Or maybe this isn't required?

Reply:
name is still that string (above).  I think category is generally different than name so I copied category from CertValidatorFactory